src/routes.js

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import { Route, IndexRoute } from 'react-router';
 import NotFoundPage from "./components/NotFoundPage";
 import HomePage from "./components/HomePage";
-import DataPackageViewContainer from "./containers/DataPackageViewContainer"; // eslint-disable-line
+import DataPackageViewContainer from "./containers/DataPackageView"; // eslint-disable-line
 import App from "./containers/App";
+import PublisherDetailsPage from "./containers/PublisherDetailsPage";
 
 export default (
 	<Route path="/" component={App}>
 		<IndexRoute component={HomePage}/>
```

Comment:
@pwalsh i will handle that new lint fixes in https://github.com/frictionlessdata/dpr-js/issues/85